src/test/java/com/basho/riak/client/core/DefaultNodeManagerTest.java

```diff
@@ -158,12 +158,13 @@ public void addNode()
         assertEquals(mockNodes.size() + 1, healthy.size());
     }
 
-    private class IsException extends ArgumentMatcher<Exception>
+    private class IsException implements ArgumentMatcher<Object>
     {
         @Override
         public boolean matches(Object argument)
         {
             return argument instanceof Exception;
```

Comment:
And then finally - we perform the test using the `instanceof` keyword - can this argument be an instance of Exception or it's subclasses.